example.js

```diff
@@ -2,14 +2,30 @@ var sub = require('./')
 var levelup = require('levelup')
 var memdown = require('memdown')
 
-var db = levelup('test', {db: memdown})
+var db = levelup(memdown())
 
-var test = sub(db, 'test', {valueEncoding: 'utf-8', separator: '@'})
-var test2 = sub(test, 'tester', {valueEncoding: 'utf-8', separator: '@'})
+var test = sub(db, 'test', { valueEncoding: 'utf-8', separator: '@' })
```

Comment:
- `utf-8` => `utf8`
- that's the default encoding though, maybe use `json` instead (esp. because the example puts an array value)
- would leave out the `separator` option, keep it simple.
- on that subject, the example is too long. Shouldn't need 3 puts, 3 gets and a read stream to demonstrate the basics.
- IMO we can remove `example.js` altogether.